src/Routers/SchemasRouter.js

```diff
@@ -14,16 +14,32 @@ function classNameMismatchResponse(bodyClass, pathClass) {
   );
 }
 
+function injectDefaultSchema(schema) {
+  let defaultSchema = Schema.defaultColumns[schema.className];
+  if (defaultSchema) {
+    Object.keys(defaultSchema).forEach((key) => {
+      schema.fields[key] = defaultSchema[key];
+    });
+  }
+  return schema;
+}
+
 function getAllSchemas(req) {
   return req.config.database.schemaCollection()
     .then(collection => collection.getAllSchemas())
+    .then(schemas => {
+      return schemas.map((schema) => {
+        return injectDefaultSchema(schema);
+      })
+    })
     .then(schemas => ({ response: { results: schemas } }));
 }
 
 function getOneSchema(req) {
   const className = req.params.className;
   return req.config.database.schemaCollection()
     .then(collection => collection.findSchema(className))
+    .then(schema => injectDefaultSchema(schema))
```

Comment:
Could be more concise here too if you feel like it:

`.then(injectDefaultSchema)`